net/remote/remote-node.sh

```diff
@@ -387,7 +380,6 @@ EOF
 
       args=(
         --url http://"$entrypointIp":8899
-        "$stake"
```

Comment:
Removing this will delegate 0.5 SOL by default.  Is that what is desired?  https://github.com/solana-labs/solana/blob/master/multinode-demo/delegate-stake.sh#L11

Reply:
wfm